gensim/test/test_fasttext.py

```diff
@@ -67,7 +70,7 @@ def setUp(self):
         self.test_new_model_file = datapath('lee_fasttext_new.bin')
 
     def test_training(self):
-        model = FT_gensim(size=10, min_count=1, hs=1, negative=0, seed=42, workers=1)
+        model = FT_gensim(size=10, min_count=1, hs=1, negative=0, seed=42, workers=1, bucket=BUCKET)
```

Comment:
I can't help feeling there's a more Pythonic way to do this, as opposed to copy-pasting this bucket size multiple times. How about something like this?

```
import functools

#
# A FT model with a limited number of buckets to save memory during testing
#
FT_gensim_toy = functools.partial(FT_gensim, bucket=5000)
```

You could also put other common parameters, like workers=1 in there as well, to reduce duplication.

model = FT_gensim_toy(...)
```

Reply:
This was my first try :) Not valid, because we also use `X.load` method and others.

```python
from gensim.model import FastText
from functools import partial

M = partial(FastText, bucket=5000)
M.load("mymodel")
```
raise
```python
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-5-db4b7c92cc06> in <module>
----> 1 M.load("mymodel")

AttributeError: 'functools.partial' object has no attribute 'load'
```